src/main/java/spoon/support/sniper/internal/AbstractSourceFragmentPrinter.java

```diff
@@ -133,29 +133,29 @@ protected void printSpaces(int fragmentIndex) {
 	 * 	false if whole `fragment` is not modified.
 	 * 	null if it is not possible to detect it here. Then it will be detected later.
```

Comment:
update comment, null value is not returned anymore?

Reply:
done, thanks.